pkg/commands/scaffoldcluster.go

```diff
@@ -0,0 +1,117 @@
+package commands
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"text/template"
+)
+
+// ScaffoldClusterOpts contains attributes used in a cluster declaration
+type ScaffoldClusterOpts struct {
+	Name string
+
+	AWSAccountID    string
+	Environment     string
+	Organization    string
+	RepositoryName  string
+	OutputDirectory string
+}
+
+// ScaffoldClusterDeclaration scaffolds a cluster declaration based on ScaffoldClusterOpts
+func ScaffoldClusterDeclaration(out io.Writer, opts ScaffoldClusterOpts) error {
+	sanitizedOpts := sanitizeOpts(opts)
+
+	t, err := template.New("cluster.yaml").Parse(clusterTemplate)
+	if err != nil {
+		return fmt.Errorf("parsing template string: %w", err)
+	}
+
+	err = t.Execute(out, sanitizedOpts)
+	if err != nil {
+		return fmt.Errorf("interpolating template: %w", err)
+	}
+
+	return nil
+}
+
+func sanitizeOpts(opts ScaffoldClusterOpts) ScaffoldClusterOpts {
+	return ScaffoldClusterOpts{
+		Name:            strings.ToLower(opts.Name),
+		AWSAccountID:    opts.AWSAccountID,
+		Environment:     strings.ToLower(opts.Environment),
+		Organization:    opts.Organization,
+		RepositoryName:  opts.RepositoryName,
+		OutputDirectory: opts.OutputDirectory,
+	}
+}
+
+const clusterTemplate = `apiVersion: okctl.io/v1alpha1
+kind: Cluster
+
+# For help finding values, see https://okctl.io/getting-started/create-cluster
+metadata:
+  # Account ID is your AWS account ID
+  accountID: '{{ .AWSAccountID }}'
+  # Environment is the name you use to identify the type of cluster it is. Common names are production, test, staging
```

Comment:
Lets put in a whitespace between each Comment:Variable pair? Will make it a lot easier to read

Reply:
@paulbes do you mean newline?

Reply:
Yeah!

Reply:
Done